Week2/homework/map-filter.js

```diff
@@ -1,8 +1,7 @@
 'use strict';
 
 function doubleOddNumbers(numbers) {
-  // Replace this comment and the next line with your code
-  console.log(numbers);
+  return numbers.filter(number => number % 2 !== 0).map(number => number * 2);
```

Comment:
Nice!

Reply:
yes, definetly. I did same branch problem in this section homeworks. I wil apply 'checkout'.